### PR TITLE
chore: enable JWPlayer's Chromecast support

### PR DESCRIPTION
**Pull request by Taxxodium**
## Description

- added configuration to enable Chromecast to JWPlayer
- added necessary UI changes to JWPlayer
- version bump

## Checklist

* [X] PR is scoped to one task
* [ ] Tests are included
* [ ] Documentation is included

* [ ] This PR is not making any UI change
* [X] This PR is making a UI change
  * [X] Screenshot(s) included

Simulator Screen Shot - iPhone 11 - 2020-01-24 at 14 56 51

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

File: Specs/JWPlayerPlugin/3.3.3/JWPlayerPlugin.podspec [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                  #  Workaround until this will be released https://github.com/CocoaPods/CocoaPods/pull/9045
+                  'OTHER_LDFLAGS' => '$(inherited) -framework "JWPlayer_iOS_SDK"',
+                  'SWIFT_VERSION' => '5.1'
+               }
+
+  s.dependency 'ZappCore'
+  s.dependency 'ApplicasterSDK'
+  s.dependency 'JWPlayer-SDK'
+  s.dependency 'GoogleAds-IMA-iOS-SDK'
+  s.dependency 'google-cast-sdk-no-bluetooth'
```
Comment (snooky23) [inline Specs/JWPlayerPlugin/3.3.3/JWPlayerPlugin.podspec]: Please work with the same cast version we are using for the generic ios plugin

File: iOS/JWPlayerPlugin.podspec [hunk 2]
```diff
@@ -27,5 +28,5 @@ Pod::Spec.new do |s|
   s.dependency 'ApplicasterSDK'
   s.dependency 'JWPlayer-SDK'
   s.dependency 'GoogleAds-IMA-iOS-SDK'
-
+  s.dependency 'google-cast-sdk-no-bluetooth'
```
Comment (snooky23) [inline iOS/JWPlayerPlugin.podspec]: Please work with the same cast version we are using for the generic ios plugin

File: iOS/Podfile [hunk 3]
```diff
@@ -12,7 +12,7 @@ target 'JWPlayerPlugin' do
   pod 'ApplicasterSDK'
   pod 'JWPlayer-SDK'
   pod 'JWPlayerPlugin', :path => 'JWPlayerPlugin.podspec'
-
+  pod 'google-cast-sdk-no-bluetooth'
```
Comment (snooky23) [inline iOS/Podfile]: Please work with the same cast version we are using for the generic ios plugin

**Review by snooky23 — CHANGES_REQUESTED** [review]
Hi, your implementation doesn't implement the base requirements of Chromecast behavior. 
1. See notes about ios 13 - https://developers.google.com/cast/docs/ios_sender/ios13_changes
2. Did you tested all Google Cast Design Checklist which can be found [here](https://developers.google.com/cast/docs/design_checklist)
3. Why are you not referring to the ZappConnector Chromecast delegate and use what we already built in the generic plugin?

Please go over the above and schedule a meeting to approve it.

**Comment by Taxxodium** [comment]
> Hi, your implementation doesn't implement the base requirements of Chromecast behavior.
> 
> 1. See notes about ios 13 - https://developers.google.com/cast/docs/ios_sender/ios13_changes

We need to define if we are going to use guest mode or not. Currently I'm using the non guest mode, but we need to discuss this.

> 2. Did you tested all Google Cast Design Checklist which can be found [here](https://developers.google.com/cast/docs/design_checklist)

JWPlayer already has support for Chromecast. From what I could see of their implementation they do comply to that checklist. But we will need to do some QA to be sure.

> 3. Why are you not referring to the ZappConnector Chromecast delegate and use what we already built in the generic plugin?

We are using JWPlayer's implementation. Just like we did with Airplay, we did not write Airplay from scratch, just enabled or disabled it in JWPlayer. Same with Chromecast.
The only thing that we do "extra" is add a button as in the screenshot above.

**Review by rkarpievich-sc — APPROVED** [review]

**Pull request closed without merging**